src/main/java/racingcar/domain/CarRacingGame.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRacingGame {
+
+    public static final int BEGIN_INDEX = 0;
+    public static final int INIT_VALUE = 0;
+    public static final String CAR_NAME_SEPARATOR = ",";
+
+    private final Cars cars;
+
+    public CarRacingGame(String carNameInput) {
+        this.cars = new Cars(makeCars(splitCarNameInput(carNameInput)));
+    }
+
+    private List<Car> makeCars(String[] carNames) {
+        List<Car> cars = new ArrayList<>();
+        for (int i = BEGIN_INDEX; i < carNames.length; i++) {
+            cars.add(new Car(carNames[i], INIT_VALUE));
+        }
+        return cars;
+    }
```

Comment:
`new Cars(makeCars(splitCarNameInput(carNameInput)));`
위와 같이 한줄에 메서드들이 나열되게 되면 가독성이 떨어진다고 생각하는데 어떻게 생각하시나요? 😄 

다음과 같이 작성했을 때와 어떤 차이가 있을까요? 😄 
```suggestion
    public CarRacingGame(String carNameInput) {
        this.cars = makeCars(carNameInput);
    }

    private Cars makeCars(String carNameInput) {
        String[] carNames = splitCarNameInput(carNameInput);

        List<Car> cars = new ArrayList<>();
        for (int i = BEGIN_INDEX; i < carNames.length; i++) {
            cars.add(new Car(carNames[i], INIT_VALUE));
        }
        return new Cars(cars);
    }
```

Reply:
코드 가독성이 향상된 것 같습니다!